requirements.txt

```diff
@@ -1,3 +1,4 @@
+dataclass-wizard
```

Comment:
I'm not convinced about adding this requirement. This project is barely maintained and only has 2 contributors.

Reply:
Yeah, true, I didn't check that.
I've replaced the lib with custom implementation of the `dict` method.